src/app/client/commitsContainer-Light.html

```diff
@@ -0,0 +1,43 @@
+<style>
+.commitStream-panel {
+  display: none;
+}
+</style>
+
+<link type="text/css" rel="stylesheet" href="css/commitStream-Light.css">
```

Comment:
Does this reference VersionOne's hosted css? We probably need to figure out if this works for shared static.

Reply:
yes